uint8[] pinNumber
---
uint8[] pinMode